.github/workflows/ci-static-sans-stdlib-build-test.yml

```diff
@@ -0,0 +1,58 @@
+name: "static-sans-stdlib-build-test-p4c"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  merge_group:
+    branches: [main]
+
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: static-sans-stdlib-build-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  # Build a p4c release on Ubuntu 20.04.
+  build-linux:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_TYPE: test
+      CMAKE_UNITY_BUILD: ON
+      BUILD_STATIC_RELEASE_SANS_STDLIB: ON
```

Comment:
Maybe even make this part of the build matrix of the static tests?

Reply:
OK, I can try to do that.